gui/pages/bundle.go

```diff
@@ -114,6 +125,80 @@ func createBundleWidget(bundle *swupd.Bundle) (*gtk.CheckButton, error) {
 	return check, nil
 }
 
+func createDecisionBox(model *model.SystemInstall, bundle *Bundle) (*decisionDialog, error) {
+	var err error
+	decisionMaker := &decisionDialog{}
+
+	decisionMaker.box, err = gtk.BoxNew(gtk.ORIENTATION_VERTICAL, 0)
+	decisionMaker.box.SetHAlign(gtk.ALIGN_FILL)
+	decisionMaker.box.SetMarginBottom(common.TopBottomMargin)
+	if err != nil {
+		return nil, err
+	}
+
+	text := utils.Locale.Get("This requires a working network connection. Proceed with a network test?")
+	decisionMaker.label, err = common.SetLabel(text, "label-warning", 0.0)
+
+	if err != nil {
+		return nil, err
+	}
+	decisionMaker.label.SetUseMarkup(true)
+	decisionMaker.label.SetHAlign(gtk.ALIGN_START)
+	decisionMaker.box.PackStart(decisionMaker.label, false, true, 0)
+
+	decisionMaker.dialog, err = common.CreateDialogOkCancel(decisionMaker.box,
+		utils.Locale.Get("Note"), utils.Locale.Get("OK"), utils.Locale.Get("CANCEL"))
```

Comment:
Please replace "Note" with "Network Required".
Change "OK" button label to "CONFIRM"

Reply:
Done